### DISCUSSION TITLE
Empty documentation pages

### DISCUSSION THREAD

**Discussion by danielzaidman** [rdkit/rdkit#4001] [General] [answered]
Dear RDKit community.
I'm a long time user, and recently many documentation page that were available before have become blank.
For example: https://rdkit.org/docs/source/rdkit.Chem.rdMolAlign.html

Was it all a dream?

**Comment by greglandrum** [comment, marked as answer]
yeah, something when wrong when I created the docs for this release. I'm currently out of the office, but I will fix it when I'm back next week.

**Reply by danielzaidman** [reply]
Thanks Greg, I appreciate this :)

**Reply by greglandrum** [reply]
This should now be resolved. Please let me know if I missed anything.
